extensions/interactions/InteractiveMap/directives/oppia-short-response-interactive-map.component.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the InteractiveMap short response.
+ *
+ * IMPORTANT NOTE: The naming convention for customization args that are passed
+ * into the directive is: the name of the parameter, followed by 'With',
+ * followed by the name of the arg.
+ */
+
+import { Component, Input, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { HtmlEscaperService } from 'services/html-escaper.service';
+
+@Component({
+  selector: 'oppia-short-response-interactive-map',
+  templateUrl: './interactive-map-short-response.component.html'
+})
+export class ShortResponseInteractiveMapComponent implements OnInit {
+  @Input() answer: string;
+  formattedCoords: string = '';
+  constructor(private htmlEscaperService: HtmlEscaperService) { }
+
+  ngOnInit(): void {
+    const _answer = this.htmlEscaperService.escapedJsonToObj(this.answer);
+    this.formattedCoords = Math.abs(_answer[0]).toFixed(3) + '° ';
+    this.formattedCoords += (_answer[0] >= 0 ? 'N' : 'S');
+    this.formattedCoords += ', ';
+    this.formattedCoords += Math.abs(_answer[1]).toFixed(3) + '° ';
+    this.formattedCoords += (_answer[1] >= 0 ? 'E' : 'W');
+  }
+}
+
+require('services/html-escaper.service.ts');
```

Comment:
Why is this here and not at the top.

Reply:
We don't need this require, removing it.